main.tf

```diff
@@ -343,3 +343,10 @@ resource "time_sleep" "wait_on_destroy" {
   # Sleep for 5 minutes to allow Karpenter to clean up resources
   destroy_duration = "5m"
 }
+
+# Store cluster name info for app deployment
+resource "aws_ssm_parameter" "cluster_name" {
+  name  = "kubernetes-platform-cluster-name"
```

Comment:
Lets add some nice namespacing. 

"platform/${local.stack_name}/cluster_name"
module.eks.cluster_name

then there can be some other namespaces

"platform/latest/cluster_name"

"platform/stable/cluster_name"

Reply:
But.. local.stack_name is the cluster name 🤔

Reply:
you also cannot have a static name like that in the module since there will be conflicts.

Do it in a way that you can use the https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/ssm_parameters_by_path so its dynamic

Reply:
I initially misused this. I updated now both PRs